src/main/java/it/pagopa/pn/common/messages/PnValidationException.java

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.pn.common.messages;
+
+import it.pagopa.pn.api.dto.notification.Notification;
+
+import javax.validation.ConstraintViolation;
+import java.util.Set;
+
+public class PnValidationException extends IllegalArgumentException {
+
+    private final Set<ConstraintViolation<Notification>> validationErrors;
```

Comment:
*Correttezza*
Sebbene ``final``, i contenuti del ``Set`` possono essere modificati: usare un ``ImmutableSet``? È in Guava di certo, magari ce ne sono anche altre implementazioni.